acc_test/data/valid_opensearch_privatelink_cluster_create.tf

```diff
@@ -0,0 +1,37 @@
+provider "instaclustr" {
+  username     = "%s"
+  api_key      = "%s"
+  api_hostname = "%s"
+}
+
+
+resource "instaclustr_cluster" "validOpenSearchPrivateLink" {
+  cluster_name            = "tf-opensearch-test"
+  data_centre             = "US_WEST_2"
+  sla_tier                = "NON_PRODUCTION"
+  cluster_network         = "192.168.0.0/18"
+  private_network_cluster = false
+  cluster_provider        = {
+    name = "AWS_VPC",
+  }
+  rack_allocation         = {
+    number_of_racks = 3
+    nodes_per_rack  = 2
+  }
+  bundle {
+    bundle  = "OPENSEARCH"
+    version = "1.3.2"
```

Comment:
this version has just been disabled when we release 1.3.4
I don't know if we have specifically enabled 1.3.2 for the tf test account, better check this

Reply:
Yeh, i will update this version